tests/VolleyManagement.UnitTests/Mvc/Controllers/GameResultsControllerTests.cs

```diff
@@ -134,6 +136,8 @@ public void CreatePostAction_InvalidModel_ExceptionThrown()
                                 .WithAwayTeamId(HOME_TEAM_ID)
                                 .Build();
 
+            gameResult.GameNumber = 5;
```

Comment:
Why did you add it here?

Reply:
In Builder I initialise GameNumber by 0, if I change in Builder to another number, two another test faild, because someone expected data with GameNumber equal 0(GameResultDtoBuilder initialize GameNumber by 0)
To make test more difficult) Remove it?

Reply:
This test verifies invalid model scenario - how game number affects it?

Reply:
Each game has a number, when user create invalid model, the error message say what to change, but we return current model to user to modify.

Reply:
Can I create new test?